app/models/service_access_token.rb

```diff
@@ -25,10 +27,16 @@ def generate
   def token
     payload = {
       iss: issuer,
-      namespace: namespace,
-      iat: Time.current.to_i,
+      iat: Time.current.to_i
     }
     payload[:sub] = subject if subject.present?
```

Comment:
what does the datastore do if the sub is empty?

Reply:
It will raise an exception here: https://github.com/ministryofjustice/fb-user-datastore/blob/master/app/controllers/user_data_controller.rb#L12
We are making sure the subject is there on the user datastore adapter class: https://github.com/ministryofjustice/fb-runner/blob/datastore/integration/app/models/user_datastore_adapter.rb#L86